source/operations/cert-manager/cert-manager-tenants.rst

```diff
@@ -0,0 +1,249 @@
+
+   Create a file called ``tenant-1-ca-certificate.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,8
+
+      # tenant-1-ca-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: tenant-1-ca-certificate
+        namespace: tenant-1
+      spec:
+        isCA: true
+        commonName: tenant-1-ca
+        secretName: tenant-1-ca-tls
+        duration: 70128h # 8y
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        issuerRef:
+          name: selfsigned-root
+          kind: ClusterIssuer
+          group: cert-manager.io
+
+3. Apply the file to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-ca-certificate.yaml
+
+2) Create the ``Issuer``
+------------------------
+
+The ``Issuer`` issues the certificates within the tenant namespace.
+
+1. Generate a resource definition for an ``Issuer``.
+
+   Create a file called ``tenant-1-ca-issuer.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+
+      # tenant-1-ca-issuer.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: tenant-1-ca-issuer
+        namespace: tenant-1
+      spec:
+        ca:
+          secretName: tenant-1-ca-tls
+
+2. Apply the ``Issuer`` resource definition to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-ca-issuer.yaml
+
+3) Create a certificate for the tenant
+--------------------------------------
+
+Request that cert-manager issue a new TLS server certificate for MinIO.
+The certificate must be valid for the following DNS domains:
+
+- ``minio.<namespace>``
+- ``minio.<namespace>.svc``
+- ``minio.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>-hl.<namespace>.svc.<cluster domain>``
+- ``*.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>.minio.<namespace>.svc.<cluster domain>'``
+
+.. important::
+
+   Replace the filler strings with values for your tenant: 
+
+   - ``<cluster domain>`` is the internal root DNS domain assigned in your Kubernetes cluster. 
+     Typically, this is ``cluster.local``, but confirm the value by checking your coredns configuration for the correct value for your Kubernetes cluster. 
+      
+     For example:
+
+     .. code-block:: shell
+        :class: copyable
+
+        kubectl get configmap coredns -n kube-system -o jsonpath="{.data}"
+
+     Different Kubernetes providers manage the root domain differently.
+     Check with your Kubernetes provider for more information.
+
+   - ``tenant-name`` is the name provided to your tenant in the ``metadata.name`` of the Tenant YAML. 
+     For this example it is ``myminio``.
+
+   - ``namespace`` is the namespace where the tenant is created, the ``metadata.namespace`` notes that in the Tenant YAML. 
```

Comment:
I'm not entirely following what this means. Do you replace `namespace` with the value you find in your Tenant YAML for `metadata.namespace`? Or does what you put here populate that value in the YAML?

Reply:
Reworked.